hazelcast-client/src/test/java/com/hazelcast/client/ClientServiceTest.java

```diff
@@ -26,14 +26,7 @@
 import com.hazelcast.client.test.TestHazelcastFactory;
 import com.hazelcast.config.Config;
 import com.hazelcast.config.ListenerConfig;
-import com.hazelcast.core.Client;
-import com.hazelcast.core.ClientListener;
-import com.hazelcast.core.ClientService;
-import com.hazelcast.core.EntryAdapter;
-import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
-import com.hazelcast.core.LifecycleEvent;
-import com.hazelcast.core.LifecycleListener;
+import com.hazelcast.core.*;
```

Comment:
@jerrinot I created a pull request for imports & unnecessary `assertTrueAllTheTime `. Can u plase review?